examples/funloc/analysis_fun.py

```diff
@@ -123,6 +123,7 @@
     gen_ssp=True,       # Generate SSP vectors
     apply_ssp=True,     # Apply SSP vectors and filtering
     plot_psd=True,      # Plot raw data power spectra
+    plot_hp=True,       # plot head movement from raw data
```

Comment:
we shouldn't need both a `params.` and a boolean here. Probably just the boolean here would be good